### PR TITLE
[10.0][MIG] users_ldap_groups

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

File: users_ldap_groups/tests/test_users_ldap_groups.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from mock import Mock, patch
+from odoo.tests.common import TransactionCase
+
+
+@patch('ldap.initialize', return_value=Mock(
+    search_st=Mock(return_value=[
+        ('cn=hello', {'name': ['hello', 'hello2']})
+    ]),
+))
```
Comment (sebalix) [inline users_ldap_groups/tests/test_users_ldap_groups.py:12]: I'm not sure but this patch call will affect the whole Python runtime, no? Would not be better to use the `start()` method in `setUp()` and `stop()` in `tearDown()`?

Comment (hbrunn) [reply users_ldap_groups/tests/test_users_ldap_groups.py:12]: https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch

Comment (sebalix) [reply users_ldap_groups/tests/test_users_ldap_groups.py:12]: Oups sorry, didn't see it was a decorator on the test class.

**Review by sebalix — APPROVED** [review]

File: users_ldap_groups/tests/test_users_ldap_groups.py (hunk 1 above)
Comment (hbrunn) [reply users_ldap_groups/tests/test_users_ldap_groups.py:12]: thanks for the review, nothing to apologize!

**Review by NL66278 — APPROVED** [review]
:smile: LGTM and tested in practice

**Pull request merged**